marimo/_utils/strings.py

```diff
@@ -84,12 +86,49 @@ def _mslex_quote(s: str) -> str:
     return quoted
 
 
+def get_short_path_name(long_name):
```

Comment:
our linting will force you to add annotations: `def get_short_path_name(long_name: str): str`